squid/conf/squid.conf

```diff
@@ -0,0 +1,55 @@
+http_port 3128
+
+maximum_object_size 100 MB
```

Comment:
Ah. This looks like we wouldn't be caching files larger than this. I think we'll have files larger. Can you find out the implications of caching larger files (besides just disk space!) - will it go slower, etc?